lib/tools/ui/layouts/column_refresher.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:io';
+import 'package:flutter/foundation.dart' show kIsWeb;
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:myecl/tools/token_expire_wrapper.dart';
+
+class ColumnRefresher extends HookConsumerWidget {
```

Comment:
I think you can go with a StatelessWidget

Reply:
I think that a ConsumerWidget is at least needed as ref is required by tokenExpireWrapper

Reply:
Nvm, I'm blind